src/test/test_rebind.py

```diff
@@ -35,6 +35,9 @@ def pick_random_port():
 
     return port
 
+if sys.hexversion < 0x02070000:
```

Comment:
Should probably also check for >=3.1 if 3.x because the empty format string field is a >=3.1 feature.

Reply:
Fixed in 22518390b0e66a4a6f9b0be12533722d62422759.